--- rt2/step7/show_mpls_table.ref
+++ rt2/step8/show_mpls_table.ref
@@ -127,22 +127,6 @@
       }
     ]
   },
-  "20301":{
-    "inLabel":20301,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
-        "nexthop":"10.12.0.1"
-      }
-    ]
-  },
   "20303":{
     "inLabel":20303,
     "installed":true,
@@ -347,22 +331,6 @@
         ],
         "distance":150,
         "installed":true,
-        "interface":"eth-rt1"
-      }
-    ]
-  },
-  "21301":{
-    "inLabel":21301,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
         "interface":"eth-rt1"
       }
     ]
