--- rt2/step7/show_ipv6_route.ref	2020-09-25 17:52:03.438020851 -0300
+++ rt2/step8/show_ipv6_route.ref	2020-09-25 17:53:21.443040633 -0300
@@ -14,10 +14,34 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-1",
+          "active":true,
+          "labels":[
+            16051,
+            16011
+          ]
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-2",
+          "active":true,
+          "labels":[
+            16051,
+            16011
+          ]
+        }
       ]
     }
   ],
@@ -36,10 +60,34 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-1",
+          "active":true,
+          "labels":[
+            16051,
+            16031
+          ]
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-2",
+          "active":true,
+          "labels":[
+            16051,
+            16031
+          ]
+        }
       ]
     }
   ],
@@ -58,6 +106,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt4-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -67,10 +118,24 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt4-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-sw1",
+          "active":true,
+          "labels":[
+            16051,
+            16041
+          ]
+        }
       ]
     }
   ],
@@ -88,19 +153,28 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
-          "active":true
+          "active":true,
+          "labels":[
+            16051
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-2",
-          "active":true
+          "active":true,
+          "labels":[
+            16051
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-1",
-          "active":true
+          "active":true,
+          "labels":[
+            16051
+          ]
         }
       ]
     }
