--- a/rt1/step4/show_ipv6_route.ref
+++ b/rt1/step5/show_ipv6_route.ref
@@ -32,18 +32,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -81,18 +70,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -130,18 +108,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
