--- a/rt1/step3/show_ip_route.ref
+++ b/rt1/step4/show_ip_route.ref
@@ -15,20 +15,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -83,20 +70,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -151,20 +125,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
