--- rt3/step3/show_mpls_table.ref	2020-08-31 22:42:48.835561429 -0300
+++ rt3/step4/show_mpls_table.ref	2020-08-31 22:42:48.839561398 -0300
@@ -7,23 +7,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.1",
-        "backupIndex":[
-          0,
-          1
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "nexthop":"10.0.5.5"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "nexthop":"10.0.4.5"
+        "nexthop":"10.0.1.1"
       }
     ]
   },
@@ -35,23 +19,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-sw1",
-        "backupIndex":[
-          0,
-          1
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "interface":"eth-rt5-2"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "interface":"eth-rt5-1"
+        "interface":"eth-sw1"
       }
     ]
   },
@@ -63,23 +31,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.2",
-        "backupIndex":[
-          0,
-          1
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "nexthop":"10.0.5.5"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "nexthop":"10.0.4.5"
+        "nexthop":"10.0.1.2"
       }
     ]
   },
@@ -91,71 +43,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-sw1",
-        "backupIndex":[
-          0,
-          1
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "interface":"eth-rt5-2"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "interface":"eth-rt5-1"
-      }
-    ]
-  },
-  "16040":{
-    "inLabel":16040,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "installed":true,
-        "nexthop":"10.0.1.2"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "installed":true,
-        "nexthop":"10.0.4.5"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "installed":true,
-        "nexthop":"10.0.5.5"
-      }
-    ]
-  },
-  "16041":{
-    "inLabel":16041,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "installed":true,
         "interface":"eth-sw1"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "installed":true,
-        "interface":"eth-rt5-1"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "installed":true,
-        "interface":"eth-rt5-2"
       }
     ]
   },
