--- a/rt3/step3/show_ipv6_route.ref
+++ b/rt3/step4/show_ipv6_route.ref
@@ -14,34 +14,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16041,
-            16011
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16041,
-            16011
-          ]
-        }
       ]
     }
   ],
@@ -60,34 +36,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16041,
-            16021
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16041,
-            16021
-          ]
-        }
       ]
     }
   ],
@@ -105,28 +57,19 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16041
-          ]
+          "active":true
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16041
-          ]
+          "active":true
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16041
-          ]
+          "active":true
         }
       ]
     }
@@ -146,9 +89,6 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt5-1",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             3
           ]
@@ -158,24 +98,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt5-2",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16041,
-            16051
-          ]
-        }
       ]
     }
   ],
