--- rt3/step3/show_ip_route.ref	2020-08-31 22:42:48.835561429 -0300
+++ rt3/step4/show_ip_route.ref	2020-08-31 22:42:48.839561398 -0300
@@ -15,34 +15,10 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040
-          ]
-        },
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040
-          ]
-        }
       ]
     }
   ],
@@ -62,34 +38,10 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040
-          ]
-        },
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040
-          ]
-        }
       ]
     }
   ],
@@ -108,30 +60,21 @@
           "ip":"10.0.1.2",
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         },
         {
           "fib":true,
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         }
       ]
     }
@@ -212,34 +155,12 @@
         {
           "ip":"10.0.1.1",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "backupIndex":[
-            0,
-            1
-          ]
+          "interfaceName":"eth-sw1"
         },
         {
           "ip":"10.0.1.2",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "backupIndex":[
-            0,
-            1
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true
-        },
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true
+          "interfaceName":"eth-sw1"
         }
       ]
     }
