--- rt3/step3/show_ip_route.ref	2020-09-25 17:48:05.506916984 -0300
+++ rt3/step4/show_ip_route.ref	2020-09-25 17:49:01.963652403 -0300
@@ -15,36 +15,10 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040,
-            16010
-          ]
-        },
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040,
-            16010
-          ]
-        }
       ]
     }
   ],
@@ -64,36 +38,10 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040,
-            16020
-          ]
-        },
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040,
-            16020
-          ]
-        }
       ]
     }
   ],
@@ -112,30 +60,21 @@
           "ip":"10.0.1.2",
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         },
         {
           "fib":true,
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         }
       ]
     }
@@ -156,9 +95,6 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             3
           ]
@@ -169,25 +105,10 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.1.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040,
-            16050
-          ]
-        }
       ]
     }
   ],
@@ -248,40 +169,12 @@
         {
           "ip":"10.0.1.1",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "backupIndex":[
-            0,
-            1
-          ]
+          "interfaceName":"eth-sw1"
         },
         {
           "ip":"10.0.1.2",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "backupIndex":[
-            0,
-            1
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.4.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "active":true,
-          "labels":[
-            16040
-          ]
-        },
-        {
-          "ip":"10.0.5.5",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "interfaceName":"eth-sw1"
         }
       ]
     }
@@ -372,30 +265,13 @@
         {
           "ip":"10.0.4.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
-          "backupIndex":[
-            0
-          ]
+          "interfaceName":"eth-rt5-1"
         },
         {
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.1.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         }
       ]
     }
@@ -411,29 +287,12 @@
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
+          "active":true
         },
         {
           "ip":"10.0.5.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.1.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "interfaceName":"eth-rt5-2"
         }
       ]
     }
@@ -528,31 +387,14 @@
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
+          "active":true
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.1.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-sw1",
-          "active":true,
-          "labels":[
-            16040
-          ]
+          "active":true
         }
       ]
     }
