--- a/rt1/step4/show_ipv6_route.ref
+++ b/rt1/step5/show_ipv6_route.ref
@@ -15,7 +15,9 @@
           "interfaceName":"eth-rt2",
           "active":true,
           "backupIndex":[
-            0
+            0,
+            1,
+            2
           ]
         }
       ],
@@ -24,6 +26,16 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
           "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true
         }
       ]
     }
@@ -44,7 +56,8 @@
           "interfaceName":"eth-rt3",
           "active":true,
           "backupIndex":[
-            0
+            0,
+            1
           ]
         }
       ],
@@ -53,6 +66,11 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
           "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
         }
       ]
     }
@@ -102,7 +120,10 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "backupIndex":[
-            0
+            0,
+            1,
+            2,
+            3
           ]
         }
       ],
@@ -111,6 +132,21 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
           "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true
         }
       ]
     }
@@ -131,7 +167,8 @@
           "interfaceName":"eth-rt6",
           "active":true,
           "backupIndex":[
-            0
+            0,
+            1
           ]
         }
       ],
@@ -140,6 +177,11 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
           "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
         }
       ]
     }
@@ -160,7 +202,10 @@
           "interfaceName":"eth-rt2",
           "active":true,
           "backupIndex":[
-            0
+            0,
+            1,
+            2,
+            3
           ]
         }
       ],
@@ -169,6 +214,21 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
           "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true
         }
       ]
     }
