--- a/rt2/step8/show_ip_route.ref
+++ b/rt2/step9/show_ip_route.ref
@@ -31,7 +31,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16050,
+            16500,
             16010
           ]
         },
@@ -41,7 +41,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16050,
+            16500,
             16010
           ]
         }
@@ -80,7 +80,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16050,
+            16500,
             16030
           ]
         },
@@ -90,7 +90,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16050,
+            16500,
             16030
           ]
         }
@@ -141,7 +141,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16050,
+            16500,
             16040
           ]
         }
@@ -165,7 +165,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         },
         {
@@ -175,7 +175,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         },
         {
@@ -185,7 +185,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         }
       ]
@@ -274,7 +274,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         },
         {
@@ -283,7 +283,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         }
       ]
@@ -321,7 +321,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         }
       ]
@@ -359,7 +359,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         }
       ]
@@ -520,7 +520,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16050
+            16500
           ]
         }
       ]
