--- a/rt2/step8/show_ipv6_route.ref
+++ b/rt2/step9/show_ipv6_route.ref
@@ -29,7 +29,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16051,
+            16501,
             16011
           ]
         },
@@ -38,7 +38,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16051,
+            16501,
             16011
           ]
         }
@@ -75,7 +75,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16051,
+            16501,
             16031
           ]
         },
@@ -84,7 +84,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16051,
+            16501,
             16031
           ]
         }
@@ -132,7 +132,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16051,
+            16501,
             16041
           ]
         }
@@ -155,7 +155,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            16051
+            16501
           ]
         },
         {
@@ -164,7 +164,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16051
+            16501
           ]
         },
         {
@@ -173,7 +173,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            16051
+            16501
           ]
         }
       ]
