--- rt3/step4/show_ip_route.ref	2020-08-31 22:42:48.839561398 -0300
+++ rt3/step5/show_ip_route.ref	2020-08-31 22:42:48.839561398 -0300
@@ -15,10 +15,34 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true,
+          "labels":[
+            16040
+          ]
+        },
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true,
+          "labels":[
+            16040
+          ]
+        }
       ]
     }
   ],
@@ -38,10 +62,34 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true,
+          "labels":[
+            16040
+          ]
+        },
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true,
+          "labels":[
+            16040
+          ]
+        }
       ]
     }
   ],
@@ -60,21 +108,30 @@
           "ip":"10.0.1.2",
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -155,12 +212,34 @@
         {
           "ip":"10.0.1.1",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1"
+          "interfaceName":"eth-sw1",
+          "backupIndex":[
+            0,
+            1
+          ]
         },
         {
           "ip":"10.0.1.2",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1"
+          "interfaceName":"eth-sw1",
+          "backupIndex":[
+            0,
+            1
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true
+        },
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true
         }
       ]
     }
