--- a/rt3/step4/show_ip_route.ref
+++ b/rt3/step5/show_ip_route.ref
@@ -15,10 +15,36 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true,
+          "labels":[
+            16040,
+            16010
+          ]
+        },
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true,
+          "labels":[
+            16040,
+            16010
+          ]
+        }
       ]
     }
   ],
@@ -38,10 +64,36 @@
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true,
+          "labels":[
+            16040,
+            16020
+          ]
+        },
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true,
+          "labels":[
+            16040,
+            16020
+          ]
+        }
       ]
     }
   ],
@@ -60,21 +112,30 @@
           "ip":"10.0.1.2",
           "afi":"ipv4",
           "interfaceName":"eth-sw1",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -95,6 +156,9 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -105,10 +169,25 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.1.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-sw1",
+          "active":true,
+          "labels":[
+            16040,
+            16050
+          ]
+        }
       ]
     }
   ],
@@ -172,12 +251,40 @@
         {
           "ip":"10.0.1.1",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1"
+          "interfaceName":"eth-sw1",
+          "backupIndex":[
+            0,
+            1
+          ]
         },
         {
           "ip":"10.0.1.2",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1"
+          "interfaceName":"eth-sw1",
+          "backupIndex":[
+            0,
+            1
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.4.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-1",
+          "active":true,
+          "labels":[
+            16040
+          ]
+        },
+        {
+          "ip":"10.0.5.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5-2",
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -268,13 +375,30 @@
         {
           "ip":"10.0.4.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-1"
+          "interfaceName":"eth-rt5-1",
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.1.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-sw1",
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -290,12 +414,29 @@
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.5.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-2"
+          "interfaceName":"eth-rt5-2",
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.1.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-sw1",
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -390,14 +531,31 @@
           "ip":"10.0.4.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.5.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.1.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-sw1",
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
