--- rt3/step4/show_mpls_table.ref	2020-09-25 17:49:00.759636711 -0300
+++ rt3/step5/show_mpls_table.ref	2020-09-25 17:50:11.280556320 -0300
@@ -7,7 +7,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.1"
+        "nexthop":"10.0.1.1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "nexthop":"10.0.4.5"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "nexthop":"10.0.5.5"
       }
     ]
   },
@@ -19,7 +35,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-sw1"
+        "interface":"eth-sw1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "interface":"eth-rt5-1"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "interface":"eth-rt5-2"
       }
     ]
   },
@@ -31,7 +63,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.2"
+        "nexthop":"10.0.1.2",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "nexthop":"10.0.4.5"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "nexthop":"10.0.5.5"
       }
     ]
   },
@@ -43,6 +91,70 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
+        "interface":"eth-sw1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "interface":"eth-rt5-1"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "interface":"eth-rt5-2"
+      }
+    ]
+  },
+  "16040":{
+    "inLabel":16040,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "installed":true,
+        "nexthop":"10.0.5.5"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "installed":true,
+        "nexthop":"10.0.4.5"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "installed":true,
+        "nexthop":"10.0.1.2"
+      }
+    ]
+  },
+  "16041":{
+    "inLabel":16041,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "installed":true,
+        "interface":"eth-rt5-2"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "installed":true,
+        "interface":"eth-rt5-1"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "installed":true,
         "interface":"eth-sw1"
       }
     ]
@@ -55,13 +167,26 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.5.5"
+        "nexthop":"10.0.5.5",
+        "backupIndex":[
+          0
+        ]
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.4.5"
+        "nexthop":"10.0.4.5",
+        "backupIndex":[
+          0
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16040,
+        "nexthop":"10.0.1.2"
       }
     ]
   },
@@ -73,13 +198,26 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt5-2"
+        "interface":"eth-rt5-2",
+        "backupIndex":[
+          0
+        ]
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt5-1"
+        "interface":"eth-rt5-1",
+        "backupIndex":[
+          0
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16041,
+        "interface":"eth-sw1"
       }
     ]
   },
