--- a/rt3/step11/show_ip_route.ref
+++ b/rt3/step12/show_ip_route.ref
@@ -198,44 +198,37 @@
       "selected":true,
       "destSelected":true,
       "distance":115,
-      "metric":30,
+      "metric":40,
       "installed":true,
       "nexthops":[
         {
           "fib":true,
-          "ip":"10.0.4.5",
+          "ip":"10.0.1.2",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-1",
+          "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
-            30060
+            16060
           ]
         },
         {
           "fib":true,
-          "ip":"10.0.5.5",
+          "ip":"10.0.4.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5-2",
+          "interfaceName":"eth-rt5-1",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             30060
           ]
-        }
-      ],
-      "backupNexthops":[
+        },
         {
-          "ip":"10.0.1.2",
+          "fib":true,
+          "ip":"10.0.5.5",
           "afi":"ipv4",
-          "interfaceName":"eth-sw1",
+          "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16060
+            30060
           ]
         }
       ]
