--- a/rt3/step11/show_mpls_table.ref
+++ b/rt3/step12/show_mpls_table.ref
@@ -165,27 +165,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30060,
-        "installed":true,
-        "nexthop":"10.0.5.5",
-        "backupIndex":[
-          0
-        ]
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30060,
-        "installed":true,
-        "nexthop":"10.0.4.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16060,
+        "installed":true,
         "nexthop":"10.0.1.2"
       }
     ]
@@ -196,27 +177,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30061,
-        "installed":true,
-        "interface":"eth-rt5-2",
-        "backupIndex":[
-          0
-        ]
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30061,
-        "installed":true,
-        "interface":"eth-rt5-1",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16061,
+        "installed":true,
         "interface":"eth-sw1"
       }
     ]
