--- rt4/step4/show_ipv6_route.ref	2020-09-25 17:49:03.399671119 -0300
+++ rt4/step5/show_ipv6_route.ref	2020-09-25 17:50:14.040592332 -0300
@@ -14,6 +14,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             16011
           ]
@@ -23,10 +26,20 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             16011
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
+        }
       ]
     }
   ],
@@ -45,6 +58,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -54,10 +70,24 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16031,
+            16021
+          ]
+        }
       ]
     }
   ],
@@ -116,10 +146,20 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true
+        }
       ]
     }
   ],
@@ -138,10 +178,20 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
+        }
       ]
     }
   ]
