--- rt4/step4/show_ip_route.ref	2020-09-25 17:49:02.163655010 -0300
+++ rt4/step5/show_ip_route.ref	2020-09-25 17:50:12.800576153 -0300
@@ -15,6 +15,9 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             16010
           ]
@@ -25,10 +28,21 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             16010
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true
+        }
       ]
     }
   ],
@@ -48,6 +62,9 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -58,10 +75,25 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16030,
+            16020
+          ]
+        }
       ]
     }
   ],
@@ -124,10 +156,21 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.7.6",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt6",
+          "active":true
+        }
       ]
     }
   ],
@@ -147,10 +190,21 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt6",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true
+        }
       ]
     }
   ],
@@ -169,13 +223,27 @@
           "ip":"10.0.2.2",
           "afi":"ipv4",
           "interfaceName":"eth-rt2-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
         },
         {
           "fib":true,
           "ip":"10.0.3.2",
           "afi":"ipv4",
           "interfaceName":"eth-rt2-2",
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -191,13 +259,30 @@
         {
           "ip":"10.0.2.2",
           "afi":"ipv4",
-          "interfaceName":"eth-rt2-1"
+          "interfaceName":"eth-rt2-1",
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.3.2",
           "afi":"ipv4",
           "interfaceName":"eth-rt2-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16030
+          ]
         }
       ]
     }
@@ -213,12 +298,29 @@
           "ip":"10.0.2.2",
           "afi":"ipv4",
           "interfaceName":"eth-rt2-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.3.2",
           "afi":"ipv4",
-          "interfaceName":"eth-rt2-2"
+          "interfaceName":"eth-rt2-2",
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16030
+          ]
         }
       ]
     }
@@ -238,6 +340,31 @@
           "ip":"10.0.6.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
+          "active":true,
+          "backupIndex":[
+            0,
+            1,
+            2
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.7.6",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt6",
+          "active":true
+        },
+        {
+          "ip":"10.0.2.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2-1",
+          "active":true
+        },
+        {
+          "ip":"10.0.3.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2-2",
           "active":true
         }
       ]
@@ -258,6 +385,31 @@
           "ip":"10.0.6.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
+          "active":true,
+          "backupIndex":[
+            0,
+            1,
+            2
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.7.6",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt6",
+          "active":true
+        },
+        {
+          "ip":"10.0.2.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2-1",
+          "active":true
+        },
+        {
+          "ip":"10.0.3.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2-2",
           "active":true
         }
       ]
@@ -273,7 +425,18 @@
         {
           "ip":"10.0.6.5",
           "afi":"ipv4",
-          "interfaceName":"eth-rt5"
+          "interfaceName":"eth-rt5",
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.7.6",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt6",
+          "active":true
         }
       ]
     }
@@ -288,7 +451,18 @@
         {
           "ip":"10.0.7.6",
           "afi":"ipv4",
-          "interfaceName":"eth-rt6"
+          "interfaceName":"eth-rt6",
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.5",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt5",
+          "active":true
         }
       ]
     }
