--- rt1/step3/show_mpls_table.ref	2020-09-25 17:48:03.782894539 -0300
+++ rt1/step4/show_mpls_table.ref	2020-09-25 17:49:00.343631290 -0300
@@ -47,30 +47,6 @@
       }
     ]
   },
-  "16040":{
-    "inLabel":16040,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "installed":true,
-        "nexthop":"10.0.1.2"
-      }
-    ]
-  },
-  "16041":{
-    "inLabel":16041,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "installed":true,
-        "interface":"eth-sw1"
-      }
-    ]
-  },
   "16050":{
     "inLabel":16050,
     "installed":true,
