--- a/rt4/step4/show_ipv6_route.ref
+++ b/rt4/step5/show_ipv6_route.ref
@@ -13,13 +13,35 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            16011
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            16011
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16011
+          ]
         }
       ]
     }
@@ -38,13 +60,36 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            3
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            3
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16031,
+            16021
+          ]
         }
       ]
     }
@@ -63,19 +108,28 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
-          "active":true
+          "active":true,
+          "labels":[
+            16031
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
-          "active":true
+          "active":true,
+          "labels":[
+            16031
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            16031
+          ]
         }
       ]
     }
@@ -94,7 +148,23 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            3
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true,
+          "labels":[
+            16051
+          ]
         }
       ]
     }
@@ -113,7 +183,23 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            3
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16061
+          ]
         }
       ]
     }
