--- a/rt4/step4/show_ipv6_route.ref
+++ b/rt4/step5/show_ipv6_route.ref
@@ -14,6 +14,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             16011
           ]
@@ -23,6 +26,19 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
+          "labels":[
+            16011
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
           "labels":[
             16011
           ]
@@ -45,6 +61,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -54,10 +73,24 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16031,
+            16021
+          ]
+        }
       ]
     }
   ],
@@ -116,10 +149,23 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
+          "active":true,
+          "labels":[
+            16051
+          ]
+        }
       ]
     }
   ],
@@ -138,10 +184,23 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true,
+          "labels":[
+            16061
+          ]
+        }
       ]
     }
   ]
