--- a/rt3/step7/show_ipv6_route.ref
+++ b/rt3/step8/show_ipv6_route.ref
@@ -148,6 +148,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         },
         {
@@ -157,6 +160,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -166,7 +172,8 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16041
+            16041,
+            30051
           ]
         }
       ]
