--- a/rt3/step7/show_ip_route.ref
+++ b/rt3/step8/show_ip_route.ref
@@ -158,6 +158,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         },
         {
@@ -168,6 +171,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -178,7 +184,8 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16040
+            16040,
+            30050
           ]
         }
       ]
