--- rt5/step5/show_mpls_table.ref	2020-09-25 17:50:11.696561748 -0300
+++ rt5/step6/show_mpls_table.ref	2020-09-25 17:51:14.685383977 -0300
@@ -1,6 +1,6 @@
 {
-  "16010":{
-    "inLabel":16010,
+  "30010":{
+    "inLabel":30010,
     "installed":true,
     "nexthops":[
       {
@@ -30,8 +30,8 @@
       }
     ]
   },
-  "16011":{
-    "inLabel":16011,
+  "30011":{
+    "inLabel":30011,
     "installed":true,
     "nexthops":[
       {
@@ -61,56 +61,56 @@
       }
     ]
   },
-  "16020":{
-    "inLabel":16020,
+  "30020":{
+    "inLabel":30020,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":16020,
         "installed":true,
-        "nexthop":"10.0.6.4"
+        "nexthop":"10.0.5.3"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16020,
         "installed":true,
-        "nexthop":"10.0.5.3"
+        "nexthop":"10.0.4.3"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16020,
         "installed":true,
-        "nexthop":"10.0.4.3"
+        "nexthop":"10.0.6.4"
       }
     ]
   },
-  "16021":{
-    "inLabel":16021,
+  "30021":{
+    "inLabel":30021,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":16021,
         "installed":true,
-        "interface":"eth-rt4"
+        "interface":"eth-rt3-2"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16021,
         "installed":true,
-        "interface":"eth-rt3-2"
+        "interface":"eth-rt3-1"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16021,
         "installed":true,
-        "interface":"eth-rt3-1"
+        "interface":"eth-rt4"
       }
     ]
   },
-  "16030":{
-    "inLabel":16030,
+  "30030":{
+    "inLabel":30030,
     "installed":true,
     "nexthops":[
       {
@@ -140,8 +140,8 @@
       }
     ]
   },
-  "16031":{
-    "inLabel":16031,
+  "30031":{
+    "inLabel":30031,
     "installed":true,
     "nexthops":[
       {
@@ -171,8 +171,8 @@
       }
     ]
   },
-  "16040":{
-    "inLabel":16040,
+  "30040":{
+    "inLabel":30040,
     "installed":true,
     "nexthops":[
       {
@@ -193,8 +193,8 @@
       }
     ]
   },
-  "16041":{
-    "inLabel":16041,
+  "30041":{
+    "inLabel":30041,
     "installed":true,
     "nexthops":[
       {
@@ -215,8 +215,8 @@
       }
     ]
   },
-  "16060":{
-    "inLabel":16060,
+  "30060":{
+    "inLabel":30060,
     "installed":true,
     "nexthops":[
       {
@@ -237,8 +237,8 @@
       }
     ]
   },
-  "16061":{
-    "inLabel":16061,
+  "30061":{
+    "inLabel":30061,
     "installed":true,
     "nexthops":[
       {
