--- a/rt6/step12/show_mpls_table.ref
+++ b/rt6/step12/show_mpls_table.ref
@@ -31,17 +31,7 @@
         "type":"SR (IS-IS)",
         "outLabel":16020,
         "installed":true,
-        "nexthop":"10.0.7.4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30020,
-        "nexthop":"10.0.8.5"
+        "nexthop":"10.0.7.4"
       }
     ]
   },
@@ -53,17 +43,7 @@
         "type":"SR (IS-IS)",
         "outLabel":16021,
         "installed":true,
-        "interface":"eth-rt4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30021,
-        "interface":"eth-rt5"
+        "interface":"eth-rt4"
       }
     ]
   },
@@ -98,17 +78,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.7.4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30040,
-        "nexthop":"10.0.8.5"
+        "nexthop":"10.0.7.4"
       }
     ]
   },
@@ -120,17 +90,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30041,
-        "interface":"eth-rt5"
+        "interface":"eth-rt4"
       }
     ]
   },