--- a/rt2/step11/show_mpls_table.ref
+++ b/rt2/step12/show_mpls_table.ref
@@ -199,7 +199,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16060,
+        "outLabel":16500,
         "nexthop":"10.0.1.3"
       }
     ]
@@ -230,7 +230,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16061,
+        "outLabel":16501,
         "interface":"eth-sw1"
       }
     ]
