--- rt6/step6/show_mpls_table.ref	2020-09-25 17:51:14.893386692 -0300
+++ rt6/step7/show_mpls_table.ref	2020-09-25 17:52:01.809999577 -0300
@@ -166,49 +166,5 @@
         "interface":"eth-rt5"
       }
     ]
-  },
-  "16050":{
-    "inLabel":16050,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "nexthop":"10.0.8.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "nexthop":"10.0.7.4"
-      }
-    ]
-  },
-  "16051":{
-    "inLabel":16051,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "interface":"eth-rt5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "interface":"eth-rt4"
-      }
-    ]
   }
 }
