--- a/rt4/step5/show_mpls_table.ref
+++ b/rt4/step6/show_mpls_table.ref
@@ -25,7 +25,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16010,
+        "outLabel":30010,
         "nexthop":"10.0.6.5"
       }
     ]
@@ -56,7 +56,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16011,
+        "outLabel":30011,
         "interface":"eth-rt5"
       }
     ]
@@ -87,7 +87,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16030,
+        "outLabel":30030,
         "nexthop":"10.0.6.5"
       }
     ]
@@ -118,7 +118,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16031,
+        "outLabel":30031,
         "interface":"eth-rt5"
       }
     ]
@@ -141,7 +141,7 @@
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16030,
+        "outLabel":30030,
         "installed":true,
         "nexthop":"10.0.6.5"
       }
@@ -165,7 +165,7 @@
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16031,
+        "outLabel":30031,
         "installed":true,
         "interface":"eth-rt5"
       }
@@ -232,7 +232,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16060,
+        "outLabel":30060,
         "nexthop":"10.0.6.5"
       }
     ]
@@ -254,7 +254,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16061,
+        "outLabel":30061,
         "interface":"eth-rt5"
       }
     ]
