--- a/rt2/step2/show_mpls_table.ref
+++ b/rt2/step3/show_mpls_table.ref
@@ -127,38 +127,6 @@
       }
     ]
   },
-  "20301":{
-    "inLabel":20301,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
-        "nexthop":"10.12.0.1"
-      }
-    ]
-  },
-  "20303":{
-    "inLabel":20303,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
-        "nexthop":"10.23.0.3"
-      }
-    ]
-  },
   "20401":{
     "inLabel":20401,
     "installed":true,
@@ -351,38 +319,6 @@
       }
     ]
   },
-  "21301":{
-    "inLabel":21301,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
-        "interface":"eth-rt1"
-      }
-    ]
-  },
-  "21303":{
-    "inLabel":21303,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "outLabelStack":[
-          3
-        ],
-        "distance":150,
-        "installed":true,
-        "interface":"eth-rt3"
-      }
-    ]
-  },
   "21401":{
     "inLabel":21401,
     "installed":true,
