--- a/rt1/step1/show_ipv6_route.ref
+++ b/rt1/step2/show_ipv6_route.ref
@@ -13,28 +13,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0,
-            1,
-            2
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
           "active":true
         }
       ]
@@ -54,22 +32,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0,
-            1
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -89,16 +51,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -118,34 +70,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true,
-          "backupIndex":[
-            0,
-            1,
-            2,
-            3
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
           "active":true
         }
       ]
@@ -165,22 +89,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
-          "active":true,
-          "backupIndex":[
-            0,
-            1
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -200,34 +108,6 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0,
-            1,
-            2,
-            3
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
           "active":true
         }
       ]
