--- a/rt1/step3/show_mpls_table.ref
+++ b/rt1/step4/show_mpls_table.ref
@@ -127,6 +127,38 @@
       }
     ]
   },
+  "20302":{
+    "inLabel":20302,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":3,
+        "outLabelStack":[
+          3
+        ],
+        "distance":150,
+        "installed":true,
+        "nexthop":"10.12.0.2"
+      }
+    ]
+  },
+  "20303":{
+    "inLabel":20303,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":20303,
+        "outLabelStack":[
+          20303
+        ],
+        "distance":150,
+        "installed":true,
+        "nexthop":"10.12.0.2"
+      }
+    ]
+  },
   "20402":{
     "inLabel":20402,
     "installed":true,
@@ -351,6 +319,38 @@
       }
     ]
   },
+  "21302":{
+    "inLabel":21302,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":3,
+        "outLabelStack":[
+          3
+        ],
+        "distance":150,
+        "installed":true,
+        "interface":"eth-rt2"
+      }
+    ]
+  },
+  "21303":{
+    "inLabel":21303,
+    "installed":true,
+    "nexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":21303,
+        "outLabelStack":[
+          21303
+        ],
+        "distance":150,
+        "installed":true,
+        "interface":"eth-rt2"
+      }
+    ]
+  },
   "21402":{
     "inLabel":21402,
     "installed":true,
