--- a/rt2/step3/show_ipv6_route.ref
+++ b/rt2/step4/show_ipv6_route.ref
@@ -14,34 +14,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-1",
-          "active":true,
-          "labels":[
-            16051,
-            16011
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-2",
-          "active":true,
-          "labels":[
-            16051,
-            16011
-          ]
-        }
       ]
     }
   ],
@@ -60,34 +36,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-1",
-          "active":true,
-          "labels":[
-            16051,
-            16031
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-2",
-          "active":true,
-          "labels":[
-            16051,
-            16031
-          ]
-        }
       ]
     }
   ],
@@ -108,9 +60,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         },
         {
@@ -120,9 +69,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
@@ -132,8 +78,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16051,
-            16041
+            16051
           ]
         }
       ]
@@ -153,10 +98,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-1",
-          "active":true,
-          "labels":[
-            16051
-          ]
+          "active":true
         },
         {
           "fib":true,
@@ -171,10 +113,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-2",
-          "active":true,
-          "labels":[
-            16051
-          ]
+          "active":true
         }
       ]
     }
@@ -196,9 +135,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            16061
           ]
         },
         {
@@ -208,9 +144,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            16061
           ]
         }
       ],
