--- a/rt4/step6/show_ipv6_route.ref
+++ b/rt4/step7/show_ipv6_route.ref
@@ -151,9 +151,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
@@ -161,10 +158,7 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
-          "active":true,
-          "labels":[
-            16051
-          ]
+          "active":true
         }
       ]
     }
