From 2054431ea3024b177083f09c66c1bb4c3d08b048 Mon Sep 17 00:00:00 2001
From: Wesley Coakley <w@wesleycoakley.com>
Date: Wed, 16 Jun 2021 00:30:50 -0400
Subject: [PATCH] don't attempt to parse dlerror() in utests

---
 tests/utests/basic/test_plugins.c | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/tests/utests/basic/test_plugins.c b/tests/utests/basic/test_plugins.c
index fd9e6130..662fd9b4 100644
--- a/tests/utests/basic/test_plugins.c
+++ b/tests/utests/basic/test_plugins.c
@@ -36,23 +36,6 @@ static void
 test_add_invalid(void **state)
 {
     assert_int_equal(LY_ESYS, lyplg_add(TESTS_BIN "/plugins/plugin_does_not_exist" LYPLG_SUFFIX));
-
-#ifdef __APPLE__
-    CHECK_LOG("Loading \""TESTS_BIN "/plugins/plugin_does_not_exist" LYPLG_SUFFIX "\" as a plugin failed "
-            "(dlopen("TESTS_BIN "/plugins/plugin_does_not_exist" LYPLG_SUFFIX ", 2): image not found).", NULL);
-#else
-    CHECK_LOG("Loading \""TESTS_BIN "/plugins/plugin_does_not_exist" LYPLG_SUFFIX "\" as a plugin failed "
-            "("TESTS_BIN "/plugins/plugin_does_not_exist" LYPLG_SUFFIX ": cannot open shared object file: "
-            "No such file or directory).", NULL);
-#endif
-
-    assert_int_equal(LY_EINVAL, lyplg_add(TESTS_BIN "/plugins/plugin_invalid" LYPLG_SUFFIX));
-#ifndef __APPLE__
-    /* OS X prints address of the symbol being searched and cmocka doesn't support wildcards in string checking assert */
-    CHECK_LOG("Processing user type plugin \""TESTS_BIN "/plugins/plugin_invalid"LYPLG_SUFFIX "\" failed, "
-            "missing type plugins information ("TESTS_BIN "/plugins/plugin_invalid"LYPLG_SUFFIX ": "
-            "undefined symbol: plugins_types__).", NULL);
-#endif
 }
 
 static void
-- 
2.31.1

