--- a/rt1/step1/show_ipv6_route.ref
+++ b/rt1/step2/show_ipv6_route.ref
@@ -13,18 +13,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -43,18 +32,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -73,18 +51,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -103,18 +70,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -133,18 +89,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -163,18 +108,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
