--- a/rt1/step1/show_ip_route.ref
+++ b/rt1/step2/show_ip_route.ref
@@ -15,20 +15,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -49,20 +36,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -83,20 +57,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -117,20 +78,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -151,20 +99,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.3",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -185,20 +120,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
