--- a/rt1/step9/show_ip_route.ref
+++ b/rt1/step10/show_ip_route.ref
@@ -15,7 +15,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -70,7 +83,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -125,7 +151,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
