--- rt5/step4/show_ipv6_route.ref	2020-09-25 17:49:03.599673726 -0300
+++ rt5/step5/show_ipv6_route.ref	2020-09-25 17:50:14.248595046 -0300
@@ -57,7 +57,10 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16021
+          ]
         },
         {
           "fib":true,
@@ -95,6 +98,9 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt3-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -104,10 +110,24 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt3-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4",
+          "active":true,
+          "labels":[
+            16021,
+            16031
+          ]
+        }
       ]
     }
   ],
@@ -128,6 +148,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
