--- rt5/step4/show_ip_route.ref	2020-09-25 17:49:02.363657616 -0300
+++ rt5/step5/show_ip_route.ref	2020-09-25 17:50:13.012578918 -0300
@@ -81,7 +81,10 @@
           "ip":"10.0.6.4",
           "afi":"ipv4",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16020
+          ]
         }
       ]
     }
@@ -102,6 +105,9 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3-1",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
@@ -112,10 +118,25 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3-2",
           "active":true,
+          "backupIndex":[
+            0
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.4",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt4",
+          "active":true,
+          "labels":[
+            16020,
+            16030
+          ]
+        }
       ]
     }
   ],
@@ -137,6 +158,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -325,13 +349,30 @@
         {
           "ip":"10.0.4.3",
           "afi":"ipv4",
-          "interfaceName":"eth-rt3-1"
+          "interfaceName":"eth-rt3-1",
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.5.3",
           "afi":"ipv4",
           "interfaceName":"eth-rt3-2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.4",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt4",
+          "active":true,
+          "labels":[
+            16020
+          ]
         }
       ]
     }
@@ -347,12 +388,29 @@
           "ip":"10.0.4.3",
           "afi":"ipv4",
           "interfaceName":"eth-rt3-1",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
         },
         {
           "ip":"10.0.5.3",
           "afi":"ipv4",
-          "interfaceName":"eth-rt3-2"
+          "interfaceName":"eth-rt3-2",
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.6.4",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt4",
+          "active":true,
+          "labels":[
+            16020
+          ]
         }
       ]
     }
