--- rt6/step5/show_mpls_table.ref	2020-08-31 22:42:48.847561334 -0300
+++ rt6/step6/show_mpls_table.ref	2020-08-31 22:42:48.847561334 -0300
@@ -5,7 +5,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16010,
+        "outLabel":30010,
         "installed":true,
         "nexthop":"10.0.8.5"
       },
@@ -23,7 +23,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16011,
+        "outLabel":30011,
         "installed":true,
         "interface":"eth-rt5"
       },
@@ -85,7 +85,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16030,
+        "outLabel":30030,
         "installed":true,
         "nexthop":"10.0.8.5",
         "backupIndex":[
@@ -107,7 +107,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16031,
+        "outLabel":30031,
         "installed":true,
         "interface":"eth-rt5",
         "backupIndex":[
