--- a/rt6/step5/show_mpls_table.ref
+++ b/rt6/step6/show_mpls_table.ref
@@ -11,7 +11,7 @@
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16010,
+        "outLabel":30010,
         "installed":true,
         "nexthop":"10.0.8.5"
       }
@@ -29,7 +29,7 @@
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16011,
+        "outLabel":30011,
         "installed":true,
         "interface":"eth-rt5"
       }
@@ -52,7 +52,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16020,
+        "outLabel":30020,
         "nexthop":"10.0.8.5"
       }
     ]
@@ -74,7 +74,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16021,
+        "outLabel":30021,
         "interface":"eth-rt5"
       }
     ]
@@ -85,7 +85,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16030,
+        "outLabel":30030,
         "installed":true,
         "nexthop":"10.0.8.5",
         "backupIndex":[
@@ -107,7 +107,7 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16031,
+        "outLabel":30031,
         "installed":true,
         "interface":"eth-rt5",
         "backupIndex":[
@@ -140,7 +140,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "nexthop":"10.0.8.5"
       }
     ]
@@ -162,7 +162,7 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "interface":"eth-rt5"
       }
     ]
