--- a/rt6/step5/show_ip_route.ref
+++ b/rt6/step6/show_ip_route.ref
@@ -26,7 +26,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16010
+            30010
           ]
         }
       ]
@@ -63,7 +63,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16020
+            30020
           ]
         }
       ]
@@ -89,7 +89,7 @@
             0
           ],
           "labels":[
-            16030
+            30030
           ]
         }
       ],
@@ -137,7 +137,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16040
+            30040
           ]
         }
       ]
