--- rt6/step5/show_ipv6_route.ref	2020-08-31 22:42:48.847561334 -0300
+++ rt6/step6/show_ipv6_route.ref	2020-08-31 22:42:48.847561334 -0300
@@ -24,7 +24,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16011
+            30011
           ]
         }
       ]
@@ -84,7 +84,7 @@
             0
           ],
           "labels":[
-            16031
+            30031
           ]
         }
       ],
