--- rt6/step5/show_ip_route.ref	2020-09-25 17:50:13.220581632 -0300
+++ rt6/step6/show_ip_route.ref	2020-09-25 17:51:16.137402938 -0300
@@ -26,7 +26,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16010
+            30010
           ]
         }
       ]
@@ -86,7 +86,7 @@
             0
           ],
           "labels":[
-            16030
+            30030
           ]
         }
       ],
