--- rt6/step5/show_ip_route.ref	2020-08-31 22:42:48.847561334 -0300
+++ rt6/step6/show_ip_route.ref	2020-08-31 22:42:48.847561334 -0300
@@ -26,7 +26,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16010
+            30010
           ]
         }
       ]
@@ -89,7 +89,7 @@
             0
           ],
           "labels":[
-            16030
+            30030
           ]
         }
       ],
