--- a/rt6/step5/show_ipv6_route.ref
+++ b/rt6/step6/show_ipv6_route.ref
@@ -24,7 +24,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16011
+            30011
           ]
         }
       ]
@@ -59,7 +59,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16021
+            30021
           ]
         }
       ]
@@ -84,7 +84,7 @@
             0
           ],
           "labels":[
-            16031
+            30031
           ]
         }
       ],
@@ -129,7 +129,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16041
+            30041
           ]
         }
       ]
