--- a/rt1/step5/show_ip_route.ref
+++ b/rt1/step6/show_ip_route.ref
@@ -15,7 +15,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -36,7 +49,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -57,7 +83,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -78,7 +117,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -99,7 +151,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt2",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.3",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
@@ -120,7 +185,20 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true
+          "onLink":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "ip":"10.0.255.2",
+          "afi":"ipv4",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "onLink":true,
+          "labels":"*"
         }
       ]
     }
