--- a/rt1/step5/show_ipv6_route.ref
+++ b/rt1/step6/show_ipv6_route.ref
@@ -13,7 +13,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
@@ -32,7 +43,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
@@ -51,7 +73,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
@@ -70,7 +103,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
@@ -89,7 +133,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
@@ -108,7 +163,18 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
-          "active":true
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true,
+          "labels":"*"
         }
       ]
     }
