--- a/rt1/step7/show_ipv6_route.ref
+++ b/rt1/step8/show_ipv6_route.ref
@@ -13,6 +13,28 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
+          "active":true,
+          "backupIndex":[
+            0,
+            1,
+            2
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
           "active":true
         }
       ]
@@ -32,6 +54,22 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
+          "active":true,
+          "backupIndex":[
+            0,
+            1
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -51,6 +89,16 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
+          "active":true,
+          "backupIndex":[
+            0
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
@@ -70,6 +118,34 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
+          "active":true,
+          "backupIndex":[
+            0,
+            1,
+            2,
+            3
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt3",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt6",
           "active":true
         }
       ]
@@ -89,6 +165,22 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt6",
+          "active":true,
+          "backupIndex":[
+            0,
+            1
+          ]
+        }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt2",
+          "active":true
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt5",
           "active":true
         }
       ]
