--- rt2/step4/show_ipv6_route.ref	2020-08-31 22:42:48.835561429 -0300
+++ rt2/step5/show_ipv6_route.ref	2020-08-31 22:42:48.835561429 -0300
@@ -14,10 +14,32 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-1",
+          "active":true,
+          "labels":[
+            16051
+          ]
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-2",
+          "active":true,
+          "labels":[
+            16051
+          ]
+        }
       ]
     }
   ],
@@ -36,10 +58,32 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
+          "backupIndex":[
+            0,
+            1
+          ],
           "labels":[
             3
           ]
         }
+      ],
+      "backupNexthops":[
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-1",
+          "active":true,
+          "labels":[
+            16051
+          ]
+        },
+        {
+          "afi":"ipv6",
+          "interfaceName":"eth-rt4-2",
+          "active":true,
+          "labels":[
+            16051
+          ]
+        }
       ]
     }
   ],
@@ -57,13 +101,19 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-1",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-2",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         }
       ]
     }
@@ -84,7 +134,7 @@
           "interfaceName":"eth-rt4-1",
           "active":true,
           "labels":[
-            3
+            16051
           ]
         },
         {
@@ -102,7 +152,7 @@
           "interfaceName":"eth-rt4-2",
           "active":true,
           "labels":[
-            3
+            16051
           ]
         }
       ]
@@ -122,13 +172,19 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-1",
-          "active":true
+          "active":true,
+          "labels":[
+            16061
+          ]
         },
         {
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4-2",
-          "active":true
+          "active":true,
+          "labels":[
+            16061
+          ]
         }
       ]
     }
