--- rt2/step1/show_ipv6_route.ref	2020-08-31 15:36:25.999825589 -0300
+++ rt2/step2/show_ipv6_route.ref	2020-08-31 22:42:48.835561429 -0300
@@ -14,32 +14,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-1",
-          "active":true,
-          "labels":[
-            16051
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-2",
-          "active":true,
-          "labels":[
-            16051
-          ]
-        }
       ]
     }
   ],
@@ -58,32 +36,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-sw1",
           "active":true,
-          "backupIndex":[
-            0,
-            1
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-1",
-          "active":true,
-          "labels":[
-            16051
-          ]
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4-2",
-          "active":true,
-          "labels":[
-            16051
-          ]
-        }
       ]
     }
   ],
