--- a/rt6/step4/show_ip_route.ref
+++ b/rt6/step5/show_ip_route.ref
@@ -14,7 +14,10 @@
           "ip":"10.0.7.4",
           "afi":"ipv4",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16010
+          ]
         },
         {
           "fib":true,
@@ -47,6 +50,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            16020
           ]
         }
       ],
@@ -92,7 +98,10 @@
           "ip":"10.0.7.4",
           "afi":"ipv4",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16030
+          ]
         }
       ]
     }
@@ -115,6 +124,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -123,7 +135,10 @@
           "ip":"10.0.8.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            16040
+          ]
         }
       ]
     }
@@ -157,7 +172,10 @@
           "ip":"10.0.7.4",
           "afi":"ipv4",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16050
+          ]
         }
       ]
     }
