--- a/rt6/step10/show_mpls_table.ref
+++ b/rt6/step11/show_mpls_table.ref
@@ -8,12 +8,6 @@
         "outLabel":16010,
         "installed":true,
         "nexthop":"10.0.7.4"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30010,
-        "installed":true,
-        "nexthop":"10.0.8.5"
       }
     ]
   },
@@ -26,12 +20,6 @@
         "outLabel":16011,
         "installed":true,
         "interface":"eth-rt4"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30011,
-        "installed":true,
-        "interface":"eth-rt5"
       }
     ]
   },
@@ -85,18 +73,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30030,
-        "installed":true,
-        "nexthop":"10.0.8.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16030,
+        "installed":true,
         "nexthop":"10.0.7.4"
       }
     ]
@@ -107,17 +85,6 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30031,
-        "installed":true,
-        "interface":"eth-rt5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16031,
         "interface":"eth-rt4"
       }
@@ -173,18 +140,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "nexthop":"10.0.8.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16500,
+        "installed":true,
         "nexthop":"10.0.7.4"
       }
     ]
@@ -195,18 +152,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "interface":"eth-rt5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16501,
+        "installed":true,
         "interface":"eth-rt4"
       }
     ]
