--- a/rt4/step3/show_mpls_table.ref
+++ b/rt4/step4/show_mpls_table.ref
@@ -7,26 +7,13 @@
         "type":"SR (IS-IS)",
         "outLabel":16010,
         "installed":true,
-        "nexthop":"10.0.3.2",
-        "backupIndex":[
-          0
-        ]
+        "nexthop":"10.0.3.2"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16010,
         "installed":true,
-        "nexthop":"10.0.2.2",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16010,
-        "nexthop":"10.0.6.5"
+        "nexthop":"10.0.2.2"
       }
     ]
   },
@@ -38,26 +25,13 @@
         "type":"SR (IS-IS)",
         "outLabel":16011,
         "installed":true,
-        "interface":"eth-rt2-2",
-        "backupIndex":[
-          0
-        ]
+        "interface":"eth-rt2-2"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":16011,
         "installed":true,
-        "interface":"eth-rt2-1",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16011,
-        "interface":"eth-rt5"
+        "interface":"eth-rt2-1"
       }
     ]
   },
@@ -69,26 +43,13 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.3.2",
-        "backupIndex":[
-          0
-        ]
+        "nexthop":"10.0.3.2"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.2.2",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16030,
-        "nexthop":"10.0.6.5"
+        "nexthop":"10.0.2.2"
       }
     ]
   },
@@ -100,26 +61,13 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt2-2",
-        "backupIndex":[
-          0
-        ]
+        "interface":"eth-rt2-2"
       },
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt2-1",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16031,
-        "interface":"eth-rt5"
+        "interface":"eth-rt2-1"
       }
     ]
   },
@@ -179,17 +127,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.6.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16050,
-        "nexthop":"10.0.7.6"
+        "nexthop":"10.0.6.5"
       }
     ]
   },
@@ -201,17 +139,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16051,
-        "interface":"eth-rt6"
+        "interface":"eth-rt5"
       }
     ]
   },
@@ -223,17 +151,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.7.6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16060,
-        "nexthop":"10.0.6.5"
+        "nexthop":"10.0.7.6"
       }
     ]
   },
@@ -245,17 +163,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16061,
-        "interface":"eth-rt5"
+        "interface":"eth-rt6"
       }
     ]
   }
