--- a/rt3/step6/show_ipv6_route.ref
+++ b/rt3/step7/show_ipv6_route.ref
@@ -148,9 +148,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         },
         {
@@ -160,9 +157,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
@@ -172,8 +166,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16041,
-            30051
+            16041
           ]
         }
       ]
