--- a/rt1/step11/show_mpls_table.ref
+++ b/rt1/step12/show_mpls_table.ref
@@ -79,12 +79,6 @@
         "type":"SR (IS-IS)",
         "outLabel":16060,
         "installed":true,
-        "nexthop":"10.0.1.3"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16060,
-        "installed":true,
         "nexthop":"10.0.1.2"
       }
     ]
@@ -96,12 +90,6 @@
       {
         "type":"SR (IS-IS)",
         "outLabel":16061,
-        "installed":true,
-        "interface":"eth-sw1"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16061,
         "installed":true,
         "interface":"eth-sw1"
       }
