--- rt2/step2/show_mpls_table.ref	2020-08-31 22:42:48.835561429 -0300
+++ rt2/step3/show_mpls_table.ref	2020-08-31 22:42:48.835561429 -0300
@@ -7,7 +7,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.1"
+        "nexthop":"10.0.1.1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16050,
+        "nexthop":"10.0.2.4"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16050,
+        "nexthop":"10.0.3.4"
       }
     ]
   },
@@ -19,7 +35,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-sw1"
+        "interface":"eth-sw1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16051,
+        "interface":"eth-rt4-1"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16051,
+        "interface":"eth-rt4-2"
       }
     ]
   },
@@ -31,7 +63,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.1.3"
+        "nexthop":"10.0.1.3",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16050,
+        "nexthop":"10.0.2.4"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16050,
+        "nexthop":"10.0.3.4"
       }
     ]
   },
@@ -43,7 +91,23 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-sw1"
+        "interface":"eth-sw1",
+        "backupIndex":[
+          0,
+          1
+        ]
+      }
+    ],
+    "backupNexthops":[
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16051,
+        "interface":"eth-rt4-1"
+      },
+      {
+        "type":"SR (IS-IS)",
+        "outLabel":16051,
+        "interface":"eth-rt4-2"
       }
     ]
   },
