--- rt3/step5/show_mpls_table.ref	2020-08-31 22:42:48.839561398 -0300
+++ rt3/step6/show_mpls_table.ref	2020-08-31 22:42:48.839561398 -0300
@@ -17,12 +17,12 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "nexthop":"10.0.5.5"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "nexthop":"10.0.4.5"
       }
     ]
@@ -45,12 +45,12 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "interface":"eth-rt5-2"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "interface":"eth-rt5-1"
       }
     ]
@@ -73,12 +73,12 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "nexthop":"10.0.5.5"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "nexthop":"10.0.4.5"
       }
     ]
@@ -101,12 +101,12 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "interface":"eth-rt5-2"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "interface":"eth-rt5-1"
       }
     ]
@@ -117,13 +117,13 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "installed":true,
         "nexthop":"10.0.4.5"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16040,
+        "outLabel":30040,
         "installed":true,
         "nexthop":"10.0.5.5"
       },
@@ -141,13 +141,13 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "installed":true,
         "interface":"eth-rt5-1"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16041,
+        "outLabel":30041,
         "installed":true,
         "interface":"eth-rt5-2"
       },
@@ -201,13 +201,13 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16060,
+        "outLabel":30060,
         "installed":true,
         "nexthop":"10.0.4.5"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16060,
+        "outLabel":30060,
         "installed":true,
         "nexthop":"10.0.5.5"
       }
@@ -219,13 +219,13 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16061,
+        "outLabel":30061,
         "installed":true,
         "interface":"eth-rt5-1"
       },
       {
         "type":"SR (IS-IS)",
-        "outLabel":16061,
+        "outLabel":30061,
         "installed":true,
         "interface":"eth-rt5-2"
       }
