--- rt3/step5/show_ip_route.ref	2020-09-25 17:50:12.592573438 -0300
+++ rt3/step6/show_ip_route.ref	2020-09-25 17:51:15.521394894 -0300
@@ -31,7 +31,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16040,
+            30040,
             16010
           ]
         },
@@ -41,7 +41,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16040,
+            30040,
             16010
           ]
         }
@@ -80,7 +80,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16040,
+            30040,
             16020
           ]
         },
@@ -90,7 +90,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16040,
+            30040,
             16020
           ]
         }
@@ -124,7 +124,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16040
+            30040
           ]
         },
         {
@@ -134,7 +134,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16040
+            30040
           ]
         }
       ]
@@ -185,7 +185,7 @@
           "active":true,
           "labels":[
             16040,
-            16050
+            30050
           ]
         }
       ]
@@ -211,7 +211,7 @@
             0
           ],
           "labels":[
-            16060
+            30060
           ]
         },
         {
@@ -224,7 +224,7 @@
             0
           ],
           "labels":[
-            16060
+            30060
           ]
         }
       ],
@@ -271,7 +271,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16040
+            30040
           ]
         },
         {
@@ -280,7 +280,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16040
+            30040
           ]
         }
       ]
