--- a/rt3/step5/show_ipv6_route.ref
+++ b/rt3/step6/show_ipv6_route.ref
@@ -29,7 +29,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16041,
+            30041,
             16011
           ]
         },
@@ -38,7 +38,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16041,
+            30041,
             16011
           ]
         }
@@ -75,7 +75,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16041,
+            30041,
             16021
           ]
         },
@@ -84,7 +84,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16041,
+            30041,
             16021
           ]
         }
@@ -107,7 +107,7 @@
           "interfaceName":"eth-rt5-1",
           "active":true,
           "labels":[
-            16041
+            30041
           ]
         },
         {
@@ -125,7 +125,7 @@
           "interfaceName":"eth-rt5-2",
           "active":true,
           "labels":[
-            16041
+            30041
           ]
         }
       ]
@@ -173,7 +173,7 @@
           "active":true,
           "labels":[
             16041,
-            16051
+            30051
           ]
         }
       ]
@@ -198,7 +198,7 @@
             0
           ],
           "labels":[
-            16061
+            30061
           ]
         },
         {
@@ -210,7 +210,7 @@
             0
           ],
           "labels":[
-            16061
+            30061
           ]
         }
       ],
