--- a/rt1/step4/show_ip_route.ref
+++ b/rt1/step5/show_ip_route.ref
@@ -36,20 +36,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -91,20 +78,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
@@ -146,20 +120,7 @@
           "afi":"ipv4",
           "interfaceName":"eth-rt3",
           "active":true,
-          "onLink":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "ip":"10.0.255.2",
-          "afi":"ipv4",
-          "interfaceName":"eth-rt2",
-          "active":true,
-          "onLink":true,
-          "labels":"*"
+          "onLink":true
         }
       ]
     }
