--- a/rt3/step6/show_ip_route.ref
+++ b/rt3/step7/show_ip_route.ref
@@ -158,9 +158,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         },
         {
@@ -171,9 +168,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
@@ -184,8 +178,7 @@
           "interfaceName":"eth-sw1",
           "active":true,
           "labels":[
-            16040,
-            30050
+            16040
           ]
         }
       ]
