--- a/rt4/step11/show_mpls_table.ref
+++ b/rt4/step12/show_mpls_table.ref
@@ -179,17 +179,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.7.6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30060,
-        "nexthop":"10.0.6.5"
+        "nexthop":"10.0.7.6"
       }
     ]
   },
@@ -201,17 +191,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":30061,
-        "interface":"eth-rt5"
+        "interface":"eth-rt6"
       }
     ]
   },
@@ -223,17 +203,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.6.5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16500,
-        "nexthop":"10.0.7.6"
+        "nexthop":"10.0.6.5"
       }
     ]
   },
@@ -245,17 +215,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt5",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16501,
-        "interface":"eth-rt6"
+        "interface":"eth-rt5"
       }
     ]
   }
