--- a/rt4/step8/show_mpls_table.ref
+++ b/rt4/step9/show_mpls_table.ref
@@ -171,15 +171,15 @@
       }
     ]
   },
-  "16050":{
-    "inLabel":16050,
+  "16060":{
+    "inLabel":16060,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.6.5",
+        "nexthop":"10.0.7.6",
         "backupIndex":[
           0
         ]
@@ -188,20 +188,20 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16050,
-        "nexthop":"10.0.7.6"
+        "outLabel":30060,
+        "nexthop":"10.0.6.5"
       }
     ]
   },
-  "16051":{
-    "inLabel":16051,
+  "16061":{
+    "inLabel":16061,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt5",
+        "interface":"eth-rt6",
         "backupIndex":[
           0
         ]
@@ -210,20 +210,20 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":16051,
-        "interface":"eth-rt6"
+        "outLabel":30061,
+        "interface":"eth-rt5"
       }
     ]
   },
-  "16060":{
-    "inLabel":16060,
+  "16500":{
+    "inLabel":16500,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.7.6",
+        "nexthop":"10.0.6.5",
         "backupIndex":[
           0
         ]
@@ -232,20 +232,20 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30060,
-        "nexthop":"10.0.6.5"
+        "outLabel":16500,
+        "nexthop":"10.0.7.6"
       }
     ]
   },
-  "16061":{
-    "inLabel":16061,
+  "16501":{
+    "inLabel":16501,
     "installed":true,
     "nexthops":[
       {
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt6",
+        "interface":"eth-rt5",
         "backupIndex":[
           0
         ]
@@ -254,8 +254,8 @@
     "backupNexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":30061,
-        "interface":"eth-rt5"
+        "outLabel":16501,
+        "interface":"eth-rt6"
       }
     ]
   }
