--- a/rt6/step3/show_ipv6_route.ref
+++ b/rt6/step4/show_ipv6_route.ref
@@ -13,10 +13,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true,
-          "labels":[
-            16011
-          ]
+          "active":true
         },
         {
           "fib":true,
@@ -47,9 +44,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            16021
           ]
         }
       ],
@@ -92,10 +86,7 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true,
-          "labels":[
-            16031
-          ]
+          "active":true
         }
       ]
     }
@@ -117,9 +108,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
@@ -127,10 +115,7 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true,
-          "labels":[
-            16041
-          ]
+          "active":true
         }
       ]
     }
@@ -162,10 +147,7 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true,
-          "labels":[
-            16051
-          ]
+          "active":true
         }
       ]
     }
