--- rt6/step3/show_mpls_table.ref	2020-09-25 17:48:04.842908340 -0300
+++ rt6/step4/show_mpls_table.ref	2020-09-25 17:49:01.363644584 -0300
@@ -7,12 +7,6 @@
         "type":"SR (IS-IS)",
         "outLabel":16010,
         "installed":true,
-        "nexthop":"10.0.7.4"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16010,
-        "installed":true,
         "nexthop":"10.0.8.5"
       }
     ]
@@ -25,12 +19,6 @@
         "type":"SR (IS-IS)",
         "outLabel":16011,
         "installed":true,
-        "interface":"eth-rt4"
-      },
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16011,
-        "installed":true,
         "interface":"eth-rt5"
       }
     ]
@@ -123,50 +111,6 @@
       }
     ]
   },
-  "16040":{
-    "inLabel":16040,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "nexthop":"10.0.7.4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "nexthop":"10.0.8.5"
-      }
-    ]
-  },
-  "16041":{
-    "inLabel":16041,
-    "installed":true,
-    "nexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "interface":"eth-rt4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":3,
-        "interface":"eth-rt5"
-      }
-    ]
-  },
   "16050":{
     "inLabel":16050,
     "installed":true,
