--- rt6/step3/show_ipv6_route.ref	2020-09-25 17:48:07.434942087 -0300
+++ rt6/step4/show_ipv6_route.ref	2020-09-25 17:49:03.847676958 -0300
@@ -22,10 +22,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true,
-          "labels":[
-            16011
-          ]
+          "active":true
         }
       ]
     }
@@ -47,9 +44,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            16021
           ]
         }
       ],
@@ -111,9 +105,6 @@
           "active":true,
           "backupIndex":[
             0
-          ],
-          "labels":[
-            3
           ]
         }
       ],
