--- a/rt1/step11/show_ipv6_route.ref
+++ b/rt1/step12/show_ipv6_route.ref
@@ -15,9 +15,7 @@
           "interfaceName":"eth-rt2",
           "active":true,
           "backupIndex":[
-            0,
-            1,
-            2
+            0
           ]
         }
       ],
@@ -26,16 +24,6 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt3",
           "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
-          "active":true
         }
       ]
     }
@@ -56,8 +44,7 @@
           "interfaceName":"eth-rt3",
           "active":true,
           "backupIndex":[
-            0,
-            1
+            0
           ]
         }
       ],
@@ -66,11 +53,6 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
           "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt5",
-          "active":true
         }
       ]
     }
@@ -120,10 +102,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "backupIndex":[
-            0,
-            1,
-            2,
-            3
+            0
           ]
         }
       ],
@@ -132,21 +111,6 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
           "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt4",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
-          "active":true
         }
       ]
     }
@@ -203,19 +167,13 @@
           "active":true,
           "backupIndex":[
             0,
-            1,
-            2
+            1
           ]
         }
       ],
       "backupNexthops":[
         {
           "afi":"ipv6",
-          "interfaceName":"eth-rt4",
-          "active":true
-        },
-        {
-          "afi":"ipv6",
           "interfaceName":"eth-rt5",
           "active":true
         },
