--- a/rt5/step11/show_mpls_table.ref
+++ b/rt5/step12/show_mpls_table.ref
@@ -179,17 +179,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "nexthop":"10.0.6.4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16040,
-        "nexthop":"10.0.8.6"
+        "nexthop":"10.0.6.4"
       }
     ]
   },
@@ -201,17 +191,7 @@
         "type":"SR (IS-IS)",
         "outLabel":3,
         "installed":true,
-        "interface":"eth-rt4",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
-        "outLabel":16041,
-        "interface":"eth-rt6"
+        "interface":"eth-rt4"
       }
     ]
   },
@@ -221,18 +201,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "nexthop":"10.0.8.6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16060,
+        "installed":true,
         "nexthop":"10.0.6.4"
       }
     ]
@@ -243,18 +213,8 @@
     "nexthops":[
       {
         "type":"SR (IS-IS)",
-        "outLabel":3,
-        "installed":true,
-        "interface":"eth-rt6",
-        "backupIndex":[
-          0
-        ]
-      }
-    ],
-    "backupNexthops":[
-      {
-        "type":"SR (IS-IS)",
         "outLabel":16061,
+        "installed":true,
         "interface":"eth-rt4"
       }
     ]
