--- a/rt5/step11/show_ipv6_route.ref
+++ b/rt5/step12/show_ipv6_route.ref
@@ -149,23 +149,10 @@
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
           "active":true,
-          "backupIndex":[
-            0
-          ],
           "labels":[
             3
           ]
         }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt6",
-          "active":true,
-          "labels":[
-            16041
-          ]
-        }
       ]
     }
   ],
@@ -176,25 +163,12 @@
       "selected":true,
       "destSelected":true,
       "distance":115,
-      "metric":20,
+      "metric":30,
       "installed":true,
       "nexthops":[
         {
           "fib":true,
           "afi":"ipv6",
-          "interfaceName":"eth-rt6",
-          "active":true,
-          "backupIndex":[
-            0
-          ],
-          "labels":[
-            3
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
           "interfaceName":"eth-rt4",
           "active":true,
           "labels":[
