--- a/rt1/step8/show_ipv6_route.ref
+++ b/rt1/step9/show_ipv6_route.ref
@@ -13,18 +13,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -73,18 +62,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
@@ -133,18 +111,7 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt2",
-          "active":true,
-          "backupIndex":[
-            0
-          ]
-        }
-      ],
-      "backupNexthops":[
-        {
-          "afi":"ipv6",
-          "interfaceName":"eth-rt3",
-          "active":true,
-          "labels":"*"
+          "active":true
         }
       ]
     }
