--- rt6/step4/show_ipv6_route.ref	2020-09-25 17:49:03.847676958 -0300
+++ rt6/step5/show_ipv6_route.ref	2020-09-25 17:50:14.456597760 -0300
@@ -22,7 +22,10 @@
           "fib":true,
           "afi":"ipv6",
           "interfaceName":"eth-rt4",
-          "active":true
+          "active":true,
+          "labels":[
+            16011
+          ]
         }
       ]
     }
@@ -44,6 +47,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            16021
           ]
         }
       ],
@@ -105,6 +111,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
