--- rt6/step4/show_ip_route.ref	2020-08-31 22:42:48.847561334 -0300
+++ rt6/step5/show_ip_route.ref	2020-08-31 22:42:48.847561334 -0300
@@ -16,7 +16,7 @@
           "interfaceName":"eth-rt4",
           "active":true,
           "labels":[
-            3
+            16010
           ]
         },
         {
@@ -50,6 +50,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            16020
           ]
         }
       ],
@@ -58,7 +61,10 @@
           "ip":"10.0.8.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         }
       ]
     }
@@ -118,6 +124,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -126,7 +135,10 @@
           "ip":"10.0.8.5",
           "afi":"ipv4",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         }
       ]
     }
