--- rt6/step4/show_ipv6_route.ref	2020-08-31 22:42:48.847561334 -0300
+++ rt6/step5/show_ipv6_route.ref	2020-08-31 22:42:48.847561334 -0300
@@ -15,7 +15,7 @@
           "interfaceName":"eth-rt4",
           "active":true,
           "labels":[
-            3
+            16011
           ]
         },
         {
@@ -47,6 +47,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            16021
           ]
         }
       ],
@@ -54,7 +57,10 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         }
       ]
     }
@@ -111,6 +117,9 @@
           "active":true,
           "backupIndex":[
             0
+          ],
+          "labels":[
+            3
           ]
         }
       ],
@@ -118,7 +127,10 @@
         {
           "afi":"ipv6",
           "interfaceName":"eth-rt5",
-          "active":true
+          "active":true,
+          "labels":[
+            3
+          ]
         }
       ]
     }
