--- a/rt4/step5/show_ipv6_route.ref
+++ b/rt4/step6/show_ipv6_route.ref
@@ -40,7 +40,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16011
+            30011
           ]
         }
       ]
@@ -87,7 +87,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16031,
+            30031,
             16021
           ]
         }
@@ -128,7 +128,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16031
+            30031
           ]
         }
       ]
@@ -198,7 +198,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16061
+            30061
           ]
         }
       ]
