--- a/rt4/step5/show_ip_route.ref
+++ b/rt4/step6/show_ip_route.ref
@@ -43,7 +43,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16010
+            30010
           ]
         }
       ]
@@ -93,7 +93,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16030,
+            30030,
             16020
           ]
         }
@@ -137,7 +137,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16030
+            30030
           ]
         }
       ]
@@ -211,7 +211,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16060
+            30060
           ]
         }
       ]
@@ -290,7 +290,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16030
+            30030
           ]
         }
       ]
@@ -328,7 +328,7 @@
           "interfaceName":"eth-rt5",
           "active":true,
           "labels":[
-            16030
+            30030
           ]
         }
       ]
